vapi/mutter-cogl-14.vapi

```diff
@@ -475,7 +475,7 @@ namespace Cogl {
 #endif
 		public void set_uniform_1f (int uniform_location, float value);
 		public void set_uniform_1i (int uniform_location, int value);
-		public void set_uniform_float (int uniform_location, int n_components, int count, float value);
+		public void set_uniform_float (int uniform_location, int n_components, int count, float *value);
```

Comment:
Is there a reason this is a pointer? According to the vapi docs if possible you should use the vala provided syntax to make the semantics clearer: https://docs.vala.dev/developer-guides/bindings/writing-a-vapi-manually/07-00-binding-a-c-function-s-parameter-and-return-types/07-07-pointers.html

Reply:
It's a pointer to the start of array of float values in C header. I don't know if there's other way to pass it

Reply:
I put it in the vapi file for now. I always assumed our vapi files were written by hand since the meson script in the vapi folder never worked for me. I guess we should ask @tintou about where should this change be